src/Rules/Install.hs

```diff
@@ -131,18 +128,18 @@ withLatestBuildStage pkg m = do
 -- Note that each time it will be recreated
 -- ref: rules/manual-package-conf.mk
 installPackageConf :: Action ()
-installPackageConf = do
+installPackageConf = do    
     let context = vanillaContext Stage0 rts
     liftIO $ IO.createDirectoryIfMissing True (takeDirectory pkgConfInstallPath)
     build $ Target context HsCpp [ pkgPath rts -/- "package.conf.in" ]
                                  [ pkgConfInstallPath <.> "raw" ]
-    Stdout out <- cmd ("grep" :: String) [ "-v", "^#pragma GCC"
-                                         , pkgConfInstallPath <.> "raw" ]
+    Stdout content <- cmd ("grep" :: String) [ "-v", "^#pragma GCC"
```

Comment:
Do you need `:: String` because of `OverloadedStrings`? If yes, how do you use the extension? (I couldn't find.)